criteria/findability.md

```diff
@@ -0,0 +1,51 @@
+# Findability
+
+## Requirements
+
+* The codebase MUST be findable using a search engine by description of the problem it solves in natural language.
+* The codebase MUST be findable using a search engine by codebase name.
+* Maintainers SHOULD submit the codebase to relevant software catalogs.
+* The codebase SHOULD have a website which describes the problem the codebase solves using the preferred jargon of different potential users of the codebase (including technologists, policy experts and managers).
+* The codebase SHOULD have a unique and persistent identifier, such as a [Wikidata item](https://www.wikidata.org/wiki/Wikidata:Introduction) or [FSF software directory listing](https://directory.fsf.org/wiki/Main_Page), mentioning the major contributors, repository location and website.
+* The codebase SHOULD include a machine-readable metadata description, for example in a [publiccode.yml](https://github.com/publiccodeyml/publiccode.yml) file.
+* There MAY be a dedicated domain name for the codebase.
```

Comment:
How does this relate to having a website, which is a SHOULD? Wouldn't having a website imply that such a domain name exists?

Reply:
@sergei-maertens consider simply having a repository well written README.md that serves the purpose, or other non-dedicated domain locations like example.org/software/foo and  example.org/software/bar ... these sites would allow a searcher to find the information, without a dedicated domain name.

Reply:
For example, a project to which I contributed has a gh-pages landing page:

https://openelectronicslab.github.io/OpenHardwareExG/

Which naturally links back to the repository itself, however there is no registered domain name like OpenHardwareExG.org or similar.